core/src/test/scala/org/broadinstitute/dsde/rawls/workspace/AggregatedWorkspaceSpec.scala

```diff
@@ -45,35 +58,35 @@ class AggregatedWorkspaceSpec extends AnyFlatSpec {
 
     val cp = ws.getCloudPlatform
 
-    cp shouldBe WorkspaceCloudPlatform.Gcp
+    cp shouldBe Some(WorkspaceCloudPlatform.Gcp)
   }
 
   it should "return Azure for an Azure MC workspace" in {
-    val ws = AggregatedWorkspace(mcWorkspace,
+    val ws = AggregatedWorkspace(readyMcWorkspace,
                                  googleProjectId = None,
                                  Some(AzureManagedAppCoordinates(UUID.randomUUID(), UUID.randomUUID(), "fake")),
                                  policies = List.empty
     )
 
     val cp = ws.getCloudPlatform
 
-    cp shouldBe WorkspaceCloudPlatform.Azure
+    cp shouldBe Some(WorkspaceCloudPlatform.Azure)
   }
 
   it should "return Gcp for a Gcp MC workspace" in {
-    val ws = AggregatedWorkspace(mcWorkspace,
+    val ws = AggregatedWorkspace(readyMcWorkspace,
                                  Some(GoogleProjectId("project-id")),
                                  azureCloudContext = None,
                                  policies = List.empty
     )
 
     val cp = ws.getCloudPlatform
-    cp shouldBe WorkspaceCloudPlatform.Gcp
+    cp shouldBe Some(WorkspaceCloudPlatform.Gcp)
   }
 
   it should "raise for an MC workspace that has cloud info for multiple clouds" in {
     val ws = AggregatedWorkspace(
-      mcWorkspace,
+      deleteFailedMcWorkspace,
```

Comment:
Checking that multiple cloud contexts throw an exception even for a non-ready workspace.